### DISCUSSION TITLE
Velo Filesystem in Wix Blocks?

### DISCUSSION THREAD

**Discussion by Cherie-R** [shoonia/velo-filesystem#23] [General]
Thank-you so much for providing this extension. I use it regularly for version control and to compare the code on my production website with a testing website. I am now working in Wix Blocks and the Download Files button is disabled. Any chance this can be enabled for Wix Blocks? :-)

**Comment by shoonia** [comment]
Hello @Cherie-R !

It's good point. I'll investigate the block to add support.

**Comment by shoonia** [comment]
Added in v0.7.0  #25

**Reply by Cherie-R** [reply]
Thank-you so much @shoonia! 👍
